using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -660,17 +748,27 @@ ACPIdentity.getExperienceCloudId { (retrievedCloudId) in
 {% tab title="React Native" %}
 #### JavaScript
 
-### getExperienceCloudId
+### getExperienceCloudId<a id="getExperienceCloudId-js"></a>
+
+This `getExperienceCloudId` API retrieves the Experience Cloud ID that was generated at initial launch  and stored in the Experience Cloud ID Service.
```

Comment:
Please reword to:
"This API retrieves the Experience Cloud ID that was generated when the app was initially launched and stored in the Experience Cloud ID Service."